### ISSUE TITLE
Long pages disappear under Footer with no scrollbar

### ISSUE THREAD

**Issue by amarkpark** [FirehoseCommunity/Lets-Pair-Program#100]
When adding a new post with the markdown editor and the preview pane (might wish to label the preview pane as "Preview" in the view since the rest of the fields are labeled) if the Message of the post is more than a line or two the "Submit" button disappears under the footer and there appears to be no scroll bar to the right to scroll down to it, one has to enlarge the browser window to see the submit button. 

See photo with no submit button and no scroll bar:

![screen shot 2016-04-20 at 1 46 41 pm](https://cloud.githubusercontent.com/assets/13741014/14686459/608a7cb0-06fe-11e6-9516-7fb118e28653.png)

**Comment by amarkpark** [comment]
Note: if I tab from the Message markdown editor field the focus does go to the submit button and I can submit the issue with a keystroke, but this is just for testing workaround, not good for end-users.

**Comment by amarkpark** [comment]
easy fix with a clear PR https://github.com/FirehoseCommunity/Lets-Pair-Program/pull/103

**Comment by risu-kun** [comment]
This is still happening for me in the master branch. What's weird is that you can make the preview box super long and it will always expand to right above the sub
![no-scroll](https://cloud.githubusercontent.com/assets/3732709/14828312/6f59fe90-0bb4-11e6-8a94-517de0ae7eb0.png)
mit button.

**Comment by amarkpark** [comment]
Yeah the clear fixed it for a moment but now it's cranky again for sure.

**Comment by amarkpark** [comment]
Seems something @cacauzen tweaked in PR https://github.com/FirehoseCommunity/Lets-Pair-Program/pull/136 has re-fixed this. if anyone else would care to verify what we're seeing and close this again that'd be great, otherwise I have a flexbox class to add to the body in the app layout that should also work nicely.
